repos/fdbt-site/src/utils/apiUtils/userData.ts

```diff
@@ -407,6 +428,114 @@ export const getReturnTicketJson = (req: NextApiRequestWithSession, res: NextApi
     };
 };
 
+export const getCappedTicketJson = async (
+    req: NextApiRequestWithSession,
+    res: NextApiResponse,
+    ticketType: 'geoZone' | 'multipleServices' | 'hybrid' | 'pointToPointPeriod',
```

Comment:
do we have | 'hybrid' | 'pointToPointPeriod' also as valid ticket type in capped user journey

Reply:
Good point, removed.